client/my-sites/site-settings/publishing-tools/index.jsx

```diff
@@ -165,7 +100,7 @@ class PublishingTools extends Component {
 		const { translate, siteIsJetpack, isAtomic } = this.props;
 
 		const renderPressThis = config.isEnabled( 'press-this' ) && ! this.isMobile();
-		const renderPostByEmail = siteIsJetpack;
+		const renderPostByEmail = true; //TODO: previously `siteIsJetpack`, is any check needed going forward?
```

Comment:
Ah, thanks for confirming. I actually meant to reevaluate that and remove it (i was pretty sure it could go) before pushing the latest changes, but I forgot I'd left myself that note. 😆